openfe/tests/utils/test_duecredit.py

```diff
@@ -0,0 +1,30 @@
+import os
+import importlib
+import pytest
+
+import openfe
+
+
+pytest.importorskip('duecredit')
+
+
+@pytest.mark.skipif((os.environ.get('DUECREDIT_ENABLE', 'yes').lower()
```

Comment:
```suggestion
@pytest.mark.skipif((os.environ.get('DUECREDIT_ENABLE', 'no').lower()
```
other way around - if you don't have the env set you should just skip the test